addOns/authhelper/src/main/resources/org/zaproxy/addon/authhelper/resources/Messages.properties

```diff
@@ -2,7 +2,22 @@ authhelper.desc = Authentication Helper
 authhelper.name = Authentication Helper
 
 authhelper.auth-detect.name = Authentication Request Identified
-authhelper.auth-detect.desc = The given request has been identified as an authentication request.\
-The 'other' field contains set of key=value lines which identify any relevant fields.\
+authhelper.auth-detect.desc = The given request has been identified as an authentication request. \
+The 'other' field contains set of key=value lines which identify any relevant fields. \
 If the request is in a context which is set to "Manual Authentication" then this rule will attempt to change the authentication to match the request identified.
 authhelper.auth-detect.soln = This is an informational alert rather than a vulnerability and so there is nothing to fix.
+
+authhelper.session.method.header.name = Header-based Session Management
+
+authhelper.session.method.header.error.value = You must specify both a header and value
+authhelper.session.method.header.error.headers = You must specify at least one header
+authhelper.session.method.header.error.json.parse = Unable to parse authentication response body from {0} as JSON: {1} 
+authhelper.session.method.header.label.header = Header
+authhelper.session.method.header.label.value = Value
+authhelper.session.method.header.label.footer = Any number of headers are supported - a new row added when the last value is entered.\n\
```

Comment:
```suggestion
authhelper.session.method.header.label.footer = Any number of headers are supported - a new row is added when the last value is completed.\n\
```

I'm not sure "completed" is any better. May "...last existing row is completed"....I dunno.,

Reply:
Have reworded - any better?